packages/runtime/src/routeGeneration/templates/express/expressTemplateService.ts

```diff
@@ -104,15 +104,16 @@ export class ExpressTemplateService extends TemplateService<ExpressApiHandlerPar
 
   protected returnHandler(params: ExpressReturnHandlerParameters) {
     const { response, statusCode, data } = params;
-    let { headers } = params;
-    headers = headers || {};
+    const { headers } = params;
 
     if (response.headersSent) {
       return;
     }
-    Object.keys(headers).forEach((name: string) => {
-      response.set(name, headers[name]);
-    });
+    if( headers ){
+      Object.keys(headers).forEach((name: string) => {
+        response.set(name, headers[name]);
+      });
+    }
```

Comment:
This if statement is also can combine with or operator like:
Object.keys(headers || {}).forEach ...

Reply:
I want avoid unecessary creation of new empty object